### PR TITLE
ONT-546 refactor error format in p2p module

### PR DESCRIPTION
**Pull request by BooniesFX**
Signed-off-by: Xiang Fu <fuxiang@onchain.com>

### CODE REVIEW

File: p2pserver/message/types/consensus_payload.go [hunk 1]
```diff
@@ -90,49 +92,75 @@ func (this *ConsensusPayload) Serialize(w io.Writer) error {
...
 
 //Serialize message payload
 func (this *ConsensusPayload) SerializeUnsigned(w io.Writer) error {
-	serialization.WriteUint32(w, this.Version)
+	err := serialization.WriteUint32(w, this.Version)
+	if err != nil {
+
+		return errors.NewDetailErr(err, errors.ErrNetPackFail, fmt.Sprintf("write error. version:%v", this.Version))
+	}
 	this.PrevHash.Serialize(w)
```
Comment (Honglei-Cong) [inline p2pserver/message/types/consensus_payload.go]: err = this.PrevHash.Serialize(w) ???

same comments for followings.

File: p2pserver/p2pserver.go [hunk 2]
```diff
@@ -468,7 +469,7 @@ func (this *P2PServer) timeout() {
 			t := p.GetContactTime()
 			if t.Before(time.Now().Add(-1 * time.Second *
 				time.Duration(periodTime) * common.KEEPALIVE_TIMEOUT)) {
-				log.Warn("Keep alive timeout!!!")
+				log.Warn("keep alive timeout!!!")
```
Comment (Honglei-Cong) [inline p2pserver/p2pserver.go]: keep alive timeout with who? from what time?

File: p2pserver/message/types/consensus_payload.go (hunk 1 above)
Comment (BooniesFX) [reply p2pserver/message/types/consensus_payload.go]: Add more check in latest code

File: p2pserver/p2pserver.go (hunk 2 above)
Comment (BooniesFX) [reply p2pserver/p2pserver.go]: add more info in latest commit

**Review by laizy — APPROVED** [review]

**Pull request merged**